src/Data/Uniques/bow.lua

```diff
@@ -220,6 +220,17 @@ Implicits: 2
 {variant:1,2,3}Gain 110% of Bow Physical Damage as Extra Damage of each Element
 {variant:4}Gain 100% of Bow Physical Damage as Extra Damage of each Element
 ]],[[
+The Gluttonous Tide
+Citadel Bow
+League: Siege of the Atlas
+Source: Drops from unique{The Eater of Worlds}
+Requires Level 58, 185 Dex
+(120-160)% increased Physical Damage
+(16-20)% increased Attack Speed
+Lose all frenzy charges on reaching maximum frenzy charges to make the next bow attack you perform fire that many additional arrows
```

Comment:
Capitalization doesn't match poedb:
`Lose all Frenzy Charges on reaching Maximum Frenzy Charges to make the next Bow Attack you perform fire that many additional Arrows`